src/git.sh

```
@@ -2,41 +1,60 @@
+git_setup_user() {
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)